apps/andi/config/integration.exs

```diff
@@ -71,7 +71,11 @@ config :andi, AndiWeb.Endpoint,
       ~r{priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$},
       ~r{priv/gettext/.*(po)$},
       ~r{lib/andi_web/controllers/.*(ex)$},
+      ~r{lib/andi_web/live/.*(ex)$},
       ~r{lib/andi_web/views/.*(ex)$},
       ~r{lib/andi_web/templates/.*(eex)$}
     ]
+  ],
+  live_view: [
+    signing_salt: "SUPER VERY TOP SECRET!!!"
```

Comment:
As part of this card can you override this in runtime.exs and make the chart changes, even if not in this PR?

Reply:
Sorry i meant to leave this on prod.exs

Reply:
We're writing a bug card for that